src/node_report.cc

```diff
@@ -782,6 +792,38 @@ static void PrintRelease(JSONWriter* writer) {
   writer->json_objectend();
 }
 
+static void PrintRequestInfo(JSONWriter* writer,
+                             Environment* env,
+                             const char* trigger) {
+  bool printOwnerInfo = strcmp(trigger, "OOMError") != 0;
+  for (ReqWrapBase* req_wrap : *env->req_wrap_queue()) {
+    AsyncWrap* w = req_wrap->GetAsyncWrap();
+    if (w->persistent().IsEmpty()) continue;
+    writer->json_start();
+    writer->json_keyvalue("type", w->MemoryInfoName().c_str());
+    if (printOwnerInfo) {
+      Local<v8::Object> owner = w->GetOwner();
+      Local<Array> keys =
+          owner->GetOwnPropertyNames(env->context()).ToLocalChecked();
+      uint32_t len = keys->Length();
+      for (uint32_t i = 0; i < len; i++) {
+        Local<Value> key = keys->Get(env->context(), i).ToLocalChecked();
+        Local<Value> value = owner->Get(env->context(), key).ToLocalChecked();
+        node::Utf8Value k(env->isolate(), key);
+        if (value->IsString()) {
+          node::Utf8Value v(env->isolate(), value);
+          writer->json_keyvalue(*k, *v);
+        } else if (value->IsNumber()) {
+          int v = static_cast<int>(value.As<v8::Uint32>()->Value());
```

Comment:
What if this is not a Uint32? What if it is a Uint32 that does not fit into the `int` range?

Reply:
It should be `double v = value.As<v8::Number>()->Value();`

Reply:
Thanks, i will update later.